src/qt/glcanvas.cpp

```diff
@@ -17,9 +17,9 @@
 #include <QtWidgets/QGestureEvent>
 
 #if defined(__VISUALC__)
-    #pragma message("OpenGL support is not implemented in wxQt")
+    #pragma message("OpenGL support is not implemented fully in wxQt")
```

Comment:
Do we still need this if OpenGL (mostly) works? I'd remove these warnings entirely.

Reply:
Removed.